ntbs-service/DataAccess/ItemRepository.cs

```diff
@@ -29,6 +30,13 @@ public async Task AddAsync(T item)
             await UpdateDatabaseAsync();
         }
 
+        public void AddWithoutSave(T item)
+        {
+            var dbSet = GetDbSet();
+            dbSet.Add(item);
+            _context.AddAuditCustomField(CustomFields.AuditDetails, NotificationAuditType.Edited);

```

Comment:
I think it's a bit strange to set a context wide property when adding a single object. I think we want to set this when we do the final save.